tests/test_batch/test_server.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+
+import sure  # noqa
+
+import moto.server as server
+from moto import mock_batch
+
+'''
+Test the different server responses
+'''
+
+@mock_batch
+def test_batch_list():
+    backend = server.create_backend_app("batch")
+    test_client = backend.test_client()
+    # do test
```

Comment:
@terrycain I think this is the only thing missing from this whole amazing PR. Let's make one call to the server to test the routing and ship shippity :shipit: